packages/legend-graph/src/graph/metamodel/pure/packageableElements/mapping/Mapping.ts

```diff
@@ -23,15 +23,18 @@ import {
   type PackageableElementVisitor,
   PackageableElement,
 } from '../PackageableElement.js';
-import type { MappingTest } from './MappingTest.js';
+import type { DEPRECATED__MappingTest } from './DEPRECATED__MappingTest.js';
 import type { MappingInclude } from './MappingInclude.js';
+import type { MappingTestSuite } from './MappingTestSuite.js';
+import type { Testable } from '../../test/Testable.js';
 
-export class Mapping extends PackageableElement implements Hashable {
+export class Mapping extends PackageableElement implements Hashable, Testable {
   includes: MappingInclude[] = [];
   classMappings: SetImplementation[] = [];
   enumerationMappings: EnumerationMapping[] = [];
   associationMappings: AssociationImplementation[] = [];
-  tests: MappingTest[] = [];
+  test: DEPRECATED__MappingTest[] = [];
+  tests: MappingTestSuite[] = [];
```

Comment:
could we rename `tests` here to `testSuites` maybe?

Reply:
keeping this in sync to the metamodel in pure